changelog/db.changelog-1.11.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="John Watson" id="1">
+    <tagDatabase tag="v1.11.0" />
+  </changeSet>
+  <changeSet author="John Watson" id="2">
+    <update tableName="batchProperties">
+      <column name="suffix" value=" (SITI_SFI)" />
+      <column name="source" value="SITI_SFI" />
+      <where>"schemeId" = 1</where>
```

Comment:
do we need to havva test case around this -- when a pilot SFI is sent through we get SITISFI whereas when a SFI 22 is sent through we get these new values?

Reply:
We already have a test case that covers the mapping is put in the correct place in the file.  This is more adding data to the tables.  The tests we have don't have access to these datasets.